### PR TITLE
Changelog for 3.12.x

### PR DESCRIPTION
**Pull request by Lex-2008**

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thanks for submitting a pull request! Maybe @craigcomstock can review this?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,53 @@
...
 Notable changes to the framework should be documented here
 
+3.12.1:
+	- Add 'system-uuid' to default dmidecode inventory (CFE-2925)
+	- Add inventory of AWS EC2 linux instances (CFE-2924)
+	- Add ubuntu 18 to package map for self upgrade (ENT-4118)
+	- Allow dmidefs inventory to be overridden via augments (CFE-2927)
+	- Also list  packages updates for hold packages: (CFE-2855)
+	- Analyze yum return code before parsing its output (CFE-2868)
+	- Avoid promise not kept (ENT-3965)
```
Comment (vpodzime) [inline CHANGELOG.md]: This needs some more details, I think.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,53 @@
...
+	- Avoid trying to read /proc/meminfo when it doesn't exist (CFE-2922)
+	- Avoid use of $(version) for package_version in legacy implementation
+	  (ENT-3963)
+	- Cleanup old report data relative to the most recent changetimestamp
+	  (ENT-4807)
+	- Configure agent_expireafter from augments (ENT-4308)
+	- Consider sles when considering suse (CFE-2897)
+	- Create desired version tracking data when necessary (ENT-3937)
```
Comment (vpodzime) [inline CHANGELOG.md]: Same here.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,53 @@
...
+	- Cron based watchdog for cf-execd on AIX (ENT-3963)
+	- Detect systemd service enablement for non native services (CFE-2932)
+	- Document how def.acl is used and how to configure it (CFE-2861)
+	- Fix compatibility with 3.7.x (ENT-3964)
```
Comment (vpodzime) [inline CHANGELOG.md]: This one can be dropped, IMHO.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,54 @@
...
+	- Make recommendations about postgresql.conf (ENT-3958)
+	- Only consider files that exist for rotation (ENT-3946)
+	- Prevent noise when a service that should be disabled is missing.
+	  (CFE-2690)
+	- Prevent standalone self upgrade from triggering un-necessarily
+	  (ENT-4092)
+	- Remove un-necessary agent run during self upgrade (ENT-4116)
+	- Specify scope => "namespace" when using persistent classes (CFE-2860)
+	- Store epoch for packages managed by zypper module (CFE-2875)
+	- Store the epoch of packages in cache db with zypper
```
Comment (olehermanse) [inline CHANGELOG.md]: "the epoch" should just be "timestamp" imo.

**Pull request merged**